docs/getting-started-guides/scratch.md

```diff
@@ -0,0 +1,658 @@
+Getting started from Scratch
+----------------------------
+
+**Table of Contents**
+
+@@TOC@@
+
+### Intended Audience
+
+This guide is for people who want to craft a custom Kubernetes cluster.  If you
+can find an existing Getting Started Guide that meets your needs on [this
+list](README.md), then we recommend using it, as you will be able to benefit
+from the experience of others.  However, if you have specific IaaS, networking,
+configuration management, or operating system requirements not met by any of
+those guides, then this guide will provide an outline of the steps you need to
+take.  Note that it requires considerably more effort than using one of the
+pre-defined guides.
+
+This guide is also useful for those wanting to understand at a high level some of the
+steps that existing cluster setup scripts are making.
+
+### Prerequisites
+  1. You should be familiar with using Kubernetes already.  We suggest you set
+    up a temporary cluster by following one of the other Getting Started Guides.
+    This will help you become familiar with the tools and API first.
+  1. You should have `kubectl` installed on your desktop.  This will happen as a side
+    effect of completing one of the other Getting Started Guides.
+
+### Selecting a Cloud Provider
+Kubernetes has the concept of a Cloud Provider, which is a module which provides
+an interface for managing TCP Load Balancers, Nodes (Intances) and Newtworking Routes.
+The interface is defined in `pkg/cloudprovider/cloud.go`.  It is possible to
+create a custom cluster without implementing a cloud provider (for example if using
+bare-metal), and not all parts of the interface need to be implemented, depending
+on how flags are set on various components. 
+
+### Selecting Nodes
+- You can use virtual or physical machines.
+- The machines need to be on the same subnet.
+- While you can build a cluster with 1 machine, in order to run all the examples and tests you
+  need at least 4 nodes.
+- Many Getting-started-guides make a distinction between the master node and regular nodes.  This
+  is not strictly necessary.
+- Nodes will need to run some version of Linux with the x86_64 architecture.  It may be possible
+  to run on other OSes and Architectures, but this guide does not try to assist with that.
+- Apiserver and etcd together are fine on a machine with 1 core and 1GB RAM for clusters with 10s of nodes.
+  Larger or more active clusters may benefit from a more cores.
+- Other nodes can have any reasonable amount of memory and any number of cores.  They need not
+  have identical configurations.
+
+### Selecting a Network
```

Comment:
@thockin please review this section on networking for:
  - blatantly false statements
  - major missing information